main.cpp

```diff
@@ -0,0 +1,111 @@
+#include <elements.hpp>
+#include <VSScript4.h>
+
+#include <string>
+
+using namespace cycfi::elements;
+
+int main(int argc, char** argv)
+{
+    try {
+        // Init variables
+        int default_frame_number{0};
+        const char* script_file_path{[&] {
+            return argc >= 2 ? argv[1] : throw std::runtime_error{"VSPreview error: Script file not specified"};
+        }()};
+        int frame_number{[&] {
+            if (argc < 3) {
+                return default_frame_number;
+            }
+            int number{std::stoi(argv[2])};
+            return number >= 0 ? number : default_frame_number;
```

Comment:
negative frame number is not a fancy way to specify 0 frame. throw an exception in such case